### PR TITLE
Set up Heroku deploy

### PR DESCRIPTION
**Pull request by dget**
With this changeset, we're set up to deploy to Heroku at http://notify-the-msc-staging.herokuapp.com/.

It will automatically deploy via Travis-CI whenever a change is pushed to master.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,12 +2,20 @@ language: python
...
   - NOTIFY_TEST_REMOTE_BROWSER=YES
 before_script:
-  - curl -L https://gist.github.com/santiycr/5139565/raw/sauce_connect_setup.sh | bash
-  - python main.py &
+- curl -L https://gist.github.com/santiycr/5139565/raw/sauce_connect_setup.sh | bash
+- python main.py &
+deploy:
+  provider: heroku
+  api_key:
+    secure: NqU8lmBBZ9T3vud0Vgo6BxnjdQxNpzyP2BHIUYiXXORaa9HIROmOhcbrSu7dcTNlT04Ux+F9eaDhHp4ybznopuCKgBcjFCi5xzahidSbudIF5yuGUOjJGE2Cwl36wbtNutQ7cOTGMLzS3mj1C2fL1ml3/1N529B0nmJCmHTozB0=
```
Comment (fureigh) [inline .travis.yml:19]: Can we use something like this? http://programmers.stackexchange.com/a/182074

Comment (dget) [inline .travis.yml:19]: That's solving a different use-case (though it's good to keep in mind). This code's for deploying to Heroku from Travis (and storing the keys in a format that Travis can access from), while that's for accessing other keys like AWS from within your Heroku app.

Comment (dget) [inline .travis.yml:19]: Note: that's not actually our API key. It's encoded in a way that Travis can decode though. As recommended per http://docs.travis-ci.com/user/deployment/heroku/

Comment (fureigh) [inline .travis.yml:19]: Okay. My real concern was, are we storing sensitive information in the repo? And you're saying we're not, correct?

Comment (fureigh) [inline .travis.yml:19]: Sensitive but encrypted. :) I'll read up on those docs; thanks for the link.

**Pull request merged**